overlord/devicestate/devicestate_test.go

```diff
@@ -801,6 +803,12 @@ func (s *deviceMgrSuite) TestDoRequestSerialErrorsOnNoHost(c *C) {
 		c.Skip("cannot run test when http proxy is in use, the error pattern is different")
 	}
 
+	// check internet access
+	_, err := net.LookupHost("nowhere.nowhere.test")
+	if err != nil && strings.Index(err.Error(), "no such host") == -1 {
+		c.Skip("cannot run test with no internet access, the error pattern is different")
+	}
+
 	privKey, _ := assertstest.GenerateKey(testKeyLength)
 
 	nowhere := "http://nowhere.nowhere.test"
```

Comment:
This constant shouldn't have the string above duplicated again here.

Reply:
Fixing this.